umm-spec-lib/test/cmr/umm_spec/test/validation/umm_spec_granule_validation_tests.clj

```diff
@@ -714,3 +714,80 @@
           :errors ["The field [End Coordinate 2] falls outside the bounds [-∞ 17] defined in the collection"]}
          {:path [:two-d-coordinate-system :start-coordinate-1]
           :errors ["The field [Start Coordinate 1] falls outside the bounds [0 ∞] defined in the collection"]}]))))
+
+(deftest granule-ocsd-equator-crossing-date-time
+  (let [collection-with-orbit (make-collection
+                               {:TemporalExtents [(cmn/map->TemporalExtentType
+                                                   {:RangeDateTimes [(helpers/range-date-time "2015-01-01T00:00:00Z" "2016-01-02T00:00:00Z")]})]
+                                :SpatialExtent
+                                {:GranuleSpatialRepresentation :orbit
+                                 :OrbitParameters {:InclinationAngle 98.2
+                                                   :Period 100.0
+                                                   :SwathWidth 2600.0
+                                                   :StartCircularLatitude 50.0
+                                                   :NumberOfOrbits 2.0}}})
+        make-granule-with-equator-crossing
```

Comment:
It's easier to read if we pull this function definition out of the let block.

Reply:
Didn't do that because this function is not generic enough, too much hard coded data just for this particular test.

Reply:
I knew exactly what you mean. I didn't do it because the granule has some fields that are tied to the collection that is defined in the `let` block. I don't want to move the function out of the `let` block to make it appear to be more independent.

Reply:
I'd just put a docstring in the extracted function saying something like "This is intended only for use by ..."

Also, it's not idiomatic Clojure to define a function in a `let`; that's what `letfn` is for ...

Reply:
OK. Moved the function out of the let block. Thanks.